builders/integrations/wallets/particle-network.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Add Particle Network to a dApp
+description: This tutorial shows you how to integrate Particle Network's Wallet-as-a-Service into a dApp built on Moonbeam to enable MPC-based onboarding and full-stack ERC-4337 AA interaction
+---
+
+# Particle Network Smart Wallet-as-a-Service
+
+
+With Particle's Smart Wallet-as-a-Service, developers can curate an unparalleled user experience through modular and customizable EOA/AA embedded wallet components. Using MPC-TSS for key management, Particle can streamline user onboarding via familiar Web2 accounts—such as Google accounts, email addresses, and phone numbers.
+
+Particle supports Moonbeam Mainnet and Testnet through both EOA interactions and native ERC-4337 SimpleAccount implementations facilitating full-stack account abstraction.
+  
+Specifically, Particle Network's Moonbeam integration is made up of a few components:
+- Particle Network Wallet-as-a-Service
+	- This is the flagship Wallet-as-a-Service product offered by Particle Network enabling application-embedded wallets powered by MPC-TSS to facilitate a seamless, Web2-like onboarding and interaction experience.
+- Particle Network Modular AA Stack
+	- Beyond base EOA-centric interaction that happens as default through Particle's Wallet-as-a-Service, Particle also has a native modular AA stack for implementation of ERC-4337 account abstraction on Moonbeam. This means inherent flexibility with the smart account, bundler, and paymaster you use in tandem with Particle's Wallet-as-a-Service when building AA-enabled applications.
+
+Currently, for AA-enabled applications, Moonbeam Mainnet and Testnet are available on Particle Network's Smart Wallet-as-a-Service through utilization of SimpleAccount. Otherwise, Moonbeam is also available directly through the standard EOA-based interface as well.
```

Comment:
What is SimpleAccount?

Reply:
SimpleAccount is the smart account implementation in which we deploy on Moonbeam to facilitate ERC4337 AA.